terasoluna-tourreservation-web/src/main/resources/META-INF/spring/spring-mvc.xml

```diff
@@ -112,6 +112,7 @@
         <property name="excludedExceptions">

```

Comment:
- [ ] Please move to before `defaultErrorView` property! (adjust same order as guideline, blank projects)